src/components/change-requests/new-change-request/new-change-request-page/common-form-fields/common-form-fields.tsx

```diff
@@ -35,7 +36,9 @@ const CommonFormFields: React.FC = () => {
       <div className={'px-4'}>
         Type
         <Form.Control as="select" custom>
-          {['New Function', 'Design Issue', 'Other', 'Initiation', 'Stage Gate'].map((t) => (
+          {Object.values(ChangeRequestType).map((
+            t // change from intiation -> Activation
```

Comment:
This comment looks like it's specific to this pull request so you'll want to remove it